.github/workflows/ansible_install_test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
```

Comment:
why is this useful?
 > pins all gh actions (except from the newrelic owned ones) to a commit hash

I mean if the intent is to improve security then we should not bump automatically hashes, no?